services/121-service/src/migrate-visa/migrate-visa.service.ts

```diff
@@ -578,4 +675,47 @@ export class MigrateVisaService {
     const linkResult = await this.httpService.post<any>(url, payload, headers);
     return linkResult;
   }
+
+  public async postTransfer(
```

Comment:
Why do we implement functions to interact with the Intersolve API here, when we also have that in the IntersolveVisaApiService? A lot of duplicate code, harder to maintain and trace bugs? (DRY)

Reply:
I just went along with how it was previously done throughout the migrate-visa file. Given that we should be done now and this is just for one-time use, it's not worth to change this anymore.

Reply:
Ok, asking @RubenGeo , since I am curious to the reason: please see my question above.